python/pyspark/sql/tests/test_session.py

```diff
@@ -225,6 +225,24 @@ def test_get_active_session_after_create_dataframe(self):
                 session2.stop()
 
 
+class SparkSessionTests5(ReusedSQLTestCase):
+
+    def test_sqlcontext_with_stopped_sparkcontext(self):
+        # SPARK-30856: test that SQLContext.getOrCreate() returns a usable instance after
+        # the SparkContext is restarted.
```

Comment:
It would be better to get the current instance of the `SQLContext` here and then later check the new context is not equal to the old.

Reply:
Done